expression/builtin_time_vec_test.go

```diff
@@ -29,7 +29,10 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.DateDiff:    {},
 	ast.TimeDiff:    {},
 	ast.DateFormat:  {},
-	ast.Hour:        {},
+	ast.Hour: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDuration}, geners: []dataGenerator{new(rangeDurationGener)}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDatetime}},
```

Comment:
Sorry I don't get it, could you explain the purpose of this line? Why may `types.ETDatetime` be an argument?

Reply:
I added the line after knowing that the default generator for `types.ETDuration` is too small to get the hours. After using `rangeDurationGener`, I think `types.ETDatetime` line is not needed. I will delete it. Do I also need to add test for null arguments?

Reply:
You can set the `rangeDurationGener.nullRation` to let it generate nulls.

Reply:
The generator will generate null values.

@qw4990 I do not understand why `ETDatetime` will not cause error...

Reply:
Is it because ETDatetime can be casted to ETDuration? vectorized `builtinCastTimeAsDurationSig` is already implemented.